src/components/filmPopups/FilmDeletePopup.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {FormEvent} from "react";
+import { Bg, BgForm, Form, BtnPopupClose, BtnSubmit } from "./filmPopups.styled";
+import { Container } from "../../styles/general";
+import {useDeleteMovieContext} from "../../context/deleteMovieContext";
+
+
+const FilmDeletePopup = () => {
+  let { isDeletePopupShown, setDeletePopupShown } = useDeleteMovieContext();
```

Comment:
If you went thus far (with abstracting `useContext()` by your custom hook), you could've gone a bit further and arrange the single context with the API similar to Redux, so that it won't require much changes once the latter will be in place. You might want to have a look for some references [over here](https://codepen.io/collection/XRMdwg).